### PR TITLE
Added header with logo, searchbar, userIcon; created nested header using CreateReactElement, JSX, Functional Component, used Composition of component

### PR DESCRIPTION
**Pull request by nitishnivedan**
*Added header with logo, searchbar, userIcon, 
* created nested header using CreateReactElement, JSX, Functional Component, used Composition of component

### CODE REVIEW

File: src/Heading/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
```
Comment (smriti-porwal) [inline src/Heading/index.js]: This import statement is not needed here. So you can remove it.

File: src/App.js [hunk 2]
```diff
@@ -1,6 +1,59 @@
 import React from "react";
```
Comment (smriti-porwal) [inline src/App.js]: This import statement is not needed here. So you can remove it.

File: src/Heading/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import usericon from "../logo/usericon.png";
+import searchicon from "../logo/searchicon.png";
+
+const Heading = ({}) => (
+  <div className="header">
+    <div className="headerComponents">
+      <img src={logo} width="50" height="50" />
+    </div>
+    <div className="searchComponentClass">
+      {" "}
```
Comment (iswarya15) [inline src/Heading/index.js:12]: just curious.. why is this {""} needed here?

**Review by iswarya15 — APPROVED** [review]

File: src/Heading/index.js (hunk 3 above)
Comment (nitishnivedan) [reply src/Heading/index.js:12]: prettier got this when I saved it. something to do with formatting then any functional impact.

File: src/Heading/index.js (hunk 1 above)
Comment (nitishnivedan) [reply src/Heading/index.js]: removed

File: src/App.js (hunk 2 above)
Comment (nitishnivedan) [reply src/App.js]: removed

**Pull request merged**